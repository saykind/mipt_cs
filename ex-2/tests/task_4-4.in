bla-bla And_then_he_said:_!!! 19
