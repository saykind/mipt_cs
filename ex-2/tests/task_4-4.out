And_then_he_said:_bla-bla!!!